src/index.d.ts

```diff
@@ -222,7 +222,7 @@ declare module 'ldclient-js' {
      * @param onDone
      *   A callback to invoke after the user is identified.
      */
-    identify: (user: LDUser, hash?: string, onDone?: () => void) => Promise<void>;
+    identify: (user: LDUser, hash?: string, onDone?: (err: Error | null, flags: LDFlagSet | null) => void) => Promise<void>;
```

Comment:
I think there's a problem here. Looking at the code, when we call `resolve` from `identify()`, the parameter is the response body we got from the requestor. That _used_ to be in the format defined by LDFlagSet (key: value), but now it is the larger response from the `evalx` endpoint (key: { value, version, etc. }). So LDFlagSet is wrong here, even though it's probably what we _should_ be returning. I'm not sure of the right solution. @apucacao?

Reply:
It's possible that non-TS users have already discovered this and are relying on  that signature. Given that, can we convert the new response to match `LDFlagSet` (if possible) and add these definitions?

Reply:
But any code that was relying on that signature would not have been working, at least not since we released summary events.